docs/source/common/debugging.rst

```diff
@@ -47,35 +53,38 @@ argument of :class:`~pytorch_lightning.trainer.trainer.Trainer`)
     loggers and logger callbacks like :class:`~pytorch_lightning.callbacks.lr_monitor.LearningRateMonitor` and
     :class:`~pytorch_lightning.callbacks.device_stats_monitor.DeviceStatsMonitor`.
 
-----------------
 
-Inspect Gradient Norms
-======================
+Shorten Epochs
+==============
 
-Logs (to a logger), the norm of each weight matrix.
+Sometimes it's helpful to only use a percentage of your training, val, test or predict data (or a set number of batches).
```

Comment:
I may say rather that in most cases it is sufficient to use just a fraction, up to 100 batches